tests/InputFiles/ScalarWave/ObserveExample.yaml

```diff
@@ -0,0 +1,53 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveScalarWave1D
+
+AnalyticSolution:
+  WaveVector: [1.0]
+  Center: [0.0]
+  Profile:
+    Sinusoid:
+      Amplitude: 1.0
+      Wavenumber: 1.0
+      Phase: 0.0
+
+InitialTime: 0.0
+FinalTime: 0.1
+InitialTimeStep: 0.001
+InitialSlabSize: 0.01
+
+DomainCreator:
+  Interval:
+    LowerBound: [0.0]
+    UpperBound: [6.283185307179586]
+    IsPeriodicIn: [true]
+    InitialRefinement: [2]
+    InitialGridPoints: [7]
+
+TimeStepper:
+  AdamsBashforthN:
+    Order: 3
+
+StepController: BinaryFraction
+
+StepChoosers:
+  - Constant: 0.05
+  - Increase:
+      Factor: 2
+  - Cfl:
+      SafetyFactor: 0.2
+
+NumericalFluxParams:
+
+# [observe_event_trigger]
+EventsAndTriggers:
+  ? EveryNSlabs:
+      N: 3
+      Offset: 5
+  : - ObserveFields:
+        VariablesToObserve: [Psi]
```

Comment:
Should we (per convention) write these in quotation marks, like the file names below?

Reply:
I suppose so.  It does seem to be what we are doing.  (What I really don't get is why we write the `./` below, but that has nothing to do with this.)

Reply:
I also don't think the `./` should be there and have long dropped it on private branches. I suppose we can remove it once someone touches those files.